controllers/message-controller.ts

```diff
@@ -35,77 +37,21 @@ export default class MessageController implements MessageControllerI {
         if(MessageController.messageController === null) {
             MessageController.messageController = new MessageController();
 
-            app.post("/api/users/:uid/messages", MessageController.messageController.sendMessage);
-            app.get("/api/users/:uid/messages", MessageController.messageController.findAllMessagesSentByUser);
-            app.get("/api/users/:uid/received-messages", MessageController.messageController.findAllMessagesReceivedByUser);
-            app.delete("/api/messages/:mid", MessageController.messageController.deleteMessage);
-            app.put("/api/messages/:mid", MessageController.messageController.updateMessage);
-            app.delete("/api/users/:uid/messages", MessageController.messageController.deleteAllMessages);
+            app.post("/api/users/:uid1/messages/:uid2", MessageController.messageService.userSendsMessage);
+            app.get("/api/users/:uid/messages", MessageController.messageService.findAllMessagesSentByUser);
+            app.get("/api/users/:uid/receives", MessageController.messageService.findAllMessagesReceivedByUser);
```

Comment:
I think we will not have any scenario where we want all the messages sent or received by the user. What might be useful is to have an API that fetches the messages sent by a user to another user. Something like `app.get("/api/users/:senderuid/messages/:receiveruid")`. This might be the best API for our chat feature

Reply:
I think we can remove these APIs since we will only be using the API `app.get("/api/users/:uid1/messages/:uid2", MessageController.messageService.findAllMessagesBetweenSpecificUsers);` according to our use case.